Makefile

```diff
@@ -1,31 +1,38 @@
-TARGETS = naive cuda cupla-cuda-async cupla-seq-seq-async cupla-seq-seq-sync cupla-tbb-seq-async cupla-omp2-seq-async kokkos-serial kokkos-openmp kokkos-cuda
+TARGETS = naive alpakaSer alpakaTBB alpakaGpuCuda cuda cupla-cuda-async cupla-seq-seq-async cupla-seq-seq-sync cupla-tbb-seq-async cupla-omp2-seq-async kokkos-serial kokkos-openmp kokkos-cuda
 
 .PHONY: all debug clean $(TARGETS)
 
-CXX := g++-8
-CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512
+EIGEN_ROOT := /data/cmssw/slc7_amd64_gcc820/external/eigen/e4c107b451c52c9ab2d7b7fa4194ee35332916ec-pafccj/include/eigen3
+TBB_ROOT := /data/cmssw/slc7_amd64_gcc820/external/tbb/2019_U3-pafccj
+BOOST :=  
+
+CXX := g++ 
+CXX_FLAGS := -O2 -std=c++14 -ftemplate-depth-512 -I/data/cmssw/slc7_amd64_gcc820/external/boost/1.67.0-pafccj/include/ -I$(TBB_ROOT)/include -L$(TBB_ROOT)/lib
```

Comment:
Why add boost?

Reply:
Boost is needed by ALPAKA.  Without boost I get this error:
```
In file included from /afs/cern.ch/work/w/wredjeb/private/external/cupla/alpaka/include/alpaka/alpaka.hpp:18,
                 from /afs/cern.ch/work/w/wredjeb/private/external/cupla/include/cupla_runtime.hpp:24,
                 from /afs/cern.ch/work/w/wredjeb/private/external/cupla/include/cuda_to_cupla.hpp:24,
                 from /afs/cern.ch/work/w/wredjeb/private/external/cupla/include/cupla/config/GpuCudaRt.hpp:33,
                 from <command-line>:
/afs/cern.ch/work/w/wredjeb/private/external/cupla/alpaka/include/alpaka/version.hpp:12:10: fatal error: boost/predef/version_number.h: No such file or directory
 #include <boost/predef/version_number.h>
````